images/component-starter-server-image/pom.xml

```diff
@@ -89,9 +89,7 @@
                   <MEECROWAVE_BASE>${docker.talend.dir.app}</MEECROWAVE_BASE>
                   <MEECROWAVE_PID>${docker.talend.dir.app}conf/server.pid</MEECROWAVE_PID>
                   <MEECROWAVE-PROPERTIES>${docker.talend.dir.app}conf/meecrowave.properties</MEECROWAVE-PROPERTIES>
-                  <TALEND_JDK_SERIAL_FILTER>oracle.sql.**;!*</TALEND_JDK_SERIAL_FILTER>
```

Comment:
I remember that we needed it for some bug...

Reply:
It was for component-server, not component-starter-server.